### PR TITLE
What do we want to do with respect to branding. 

### PR DESCRIPTION
**Pull request by balhar-jakub**

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]

File: Technical-Steering-Committee/branding.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Branding 
+
+All the projects by default use the default Zowe logo and imagery. The projects don't create derivatives
```
Comment (balhar-jakub) [inline Technical-Steering-Committee/branding.md:3]: Provide link to the OMP policies around the Zowe rules. 
Add few more links.

File: Technical-Steering-Committee/branding.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Branding 
+
+This policy builds upon and extends the [Linux Foundation trademark policy](https://www.linuxfoundation.org/legal/trademark-usage) and [Open Mainframe Project's Branding guidelines](https://www.openmainframeproject.org/branding-guidelines).
+
+All the projects by default use the Zowe logo and imagery approved by TSC. The projects don't create derivatives
```
Comment (1000TurquoisePogs) [inline Technical-Steering-Committee/branding.md:5]: The wording is strange to me for a policy document. "all projects by default use" implies a fact that is desired but may not be true. To make that desire a policy, how about:

"All projects must use the [standard Zowe logo](https://artwork.openmainframeproject.org/projects/zowe/) ([github source](https://github.com/openmainframeproject/artwork/tree/main/projects/zowe)) and imagery approved by the Technical Steering Committee (TSC), Open Mainframe Project, and Linux Foundation. Projects must not create derivatives or custom logos except when granted through an exception by the TSC. Such exception will be granted in cooperation with the Linux Foundation [art team](https://github.com/openmainframeproject/artwork).

Comment (balhar-jakub) [reply Technical-Steering-Committee/branding.md]: Accepted your version.

**Review by 1000TurquoisePogs — APPROVED** [review]

**Comment by jmertic** [comment]
This looks good - you might want to add references to the LF Trademark Policy ( https://www.linuxfoundation.org/trademark-usage ) and Open Mainframe Project Branding Guidelines ( https://www.openmainframeproject.org/branding-guidelines ) for instructions on proper usage.

File: Technical-Steering-Committee/branding.md
Comment (jmertic) [inline Technical-Steering-Committee/branding.md:1]: Note that we'd need the LF and Open Mainframe Project approval on derivatives and custom logos as well

**Pull request merged**